README.md

```diff
@@ -1,147 +1,184 @@
 
 ### Linux
 
+Run the below command to install `binutils-dev` package on Linux.
+
 ```bash
 sudo apt install binutils-dev
 ```
 
 ### Windows
-Install Visual Studio (MSVC), for example the version 2022, you can download the
-Community version for free from: https://visualstudio.microsoft.com/downloads/.
 
-Launch the Miniforge prompt from the Desktop.
+Please follow the below steps for Windows:
 
-In the shell, initialize the MSVC compiler using:
+- Install Visual Studio, for example the version 2022.
 
-```bash
-call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd" -arch=x64
-```
+  - You can download the
+    Community version for free from: https://visualstudio.microsoft.com/downloads/.
+  - After installing Visual Studio and running the Visual Studio Installer, you must install the "Desktop Development with C++" workload which will install Visual C++ Compiler (MSVC).
 
-You can optionally test MSVC via:
+- Launch the Miniforge prompt from the Desktop.
 
-```bash
-cl /?
-link /?
-```
+  - It is recommended to use MiniForge instead of Powershell as the main terminal to build and write code for LPython.
 
-Both commands must print several pages of help text.
+- In the MiniForge Prompt, initialize the MSVC compiler using the below command:
 
-## Build LPython
+  ```bash
+  call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd" -arch=x64
+  ```
 
-Clone LPython
+- You can optionally test MSVC via:
 
-```bash
-git clone https://github.com/lcompilers/lpython.git
-cd lpython
-```
+  ```bash
+  cl /?
+  link /?
+  ```
+
+  Both commands must print several pages of help text.
+
+## Step 3: Build LPython
+
+- Clone LPython using the following commands
+
+  ```bash
+  git clone https://github.com/lcompilers/lpython.git
+  cd lpython
+  ```
+
+  You may also use GitHub Desktop to do the same.
 
 ### Linux and MacOS
 
 - Create a Conda environment using the pre-existing file:
 
-```bash
-conda env create -f environment_unix.yml
-conda activate lp
-```
+  ```bash
+  conda env create -f environment_unix.yml
+  conda activate lp
+  ```
 
 - Generate prerequisite files; build in Debug Mode:
 
-```bash
-./build0.sh
-./build1.sh
-```
+  ```bash
+  ./build0.sh
+  ./build1.sh
+  ```
 
 ### Windows
 
 - Create a Conda environment using the pre-existing file:
 
-```bash
-conda env create -f environment_win.yml
-conda activate lp
-```
+  ```bash
+  conda env create -f environment_win.yml
+  conda activate lp
+  ```
 
 - Generate prerequisite files; build in Release Mode:
 
-```bash
-call build0.bat
-call build1.bat
-```
+  ```bash
+  call build0.bat
+  call build1.bat
+  ```
 
-- Tests and examples
+- Tests and examples (Optional)
 
-```bash
-ctest
-inst\bin\lpython examples\expr2.py
-inst\bin\lpython examples\expr2.py -o a.out
-a.out
-```
+  ```bash
+  ctest
+  inst\bin\lpython examples\expr2.py
+  inst\bin\lpython examples\expr2.py -o a.out
+  a.out
```

Comment:
This is a basic check that everything works after the build, @khairalanam did you face any issues in running this command?
P.S. I ask this because you have added this as Optional.

Reply:
I ran the command before but I didn't face issues. But once I tried building LPython from scratch again, I forgot to run this command and then realized I haven't faced any issues in running LPython due to not running this command. For the record, I followed all the steps until and including the step to building the prerequisite files which is where I got stuck a couple of times.

I will remove the "Optional" tag because as you said it is a basic check to see if everything is working.